--- a/webpack.common.mjs
+++ b/webpack.common.mjs
@@ -30,7 +30,7 @@ export const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
 const tsConfig = JSON.parse(await readFile("./tsconfig.json"));
 const tauriConf = JSON.parse(await readFile("./src-tauri/tauri.conf.json"));

-const gitVersion = process.env.TRGUING_VERSION ?? execaSync("git", ["describe", "--tags", "--dirty", "--always"]).stdout;
+const gitVersion = process.env.TRGUING_VERSION ?? execaSync("git", ["describe", "--tags", "--always"]).stdout;
 const versionInfo = `{
     "gitVersion": "${gitVersion}",
     "backendVersion": "${tauriConf.version}",
